code/modules/jobs/job_types/assistant.dm

```diff
@@ -21,6 +21,14 @@ Assistant
 
 	family_heirlooms = list(/obj/item/storage/toolbox/mechanical/old/heirloom, /obj/item/clothing/gloves/cut/heirloom)
 
+	mail_goodies = list(
+		/obj/effect/spawner/lootdrop/donkpockets = 10,
+		/obj/item/clothing/mask/gas = 10,
+		/obj/item/clothing/gloves/color/fyellow = 7,
```

Comment:
additional suggestion for this list: toolbelts and red crowbars

Reply:
how about large non-pocket crowbars, as a novelty?